crates/driver/src/domain/competition/solution/mod.rs

```diff
@@ -133,32 +133,46 @@ impl Solution {
         self.gas
     }
 
+    fn trade_count_for_scoring(&self, trade: &Trade) -> bool {
+        match trade {
+            Trade::Fulfillment(_) => true,
```

Comment:
liquidity orders also should not generate any surplus. We no longer create these so technically it doesn't matter but it would be better to cover this case explicitly to indicate that we didn't forget about it.

Reply:
@MartinquaXD  Oh, damn, you are right, sorry :pray: